lms/templates/footer.html

```diff
@@ -83,3 +83,19 @@
     <link rel="stylesheet" type="text/css" href="${url}"></link>
   % endfor
 % endif
+
+% if settings.FEATURES.get('ENABLE_COOKIE_CONSENT', False):
+<!-- Cookie Consent -->
+<link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.css" />
+<script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.js"></script>
+<script>
+  window.cookieconsent.initialise({
```

Comment:
@andy-armstrong good point re i18n/RTL. Will get both sorted as soon as possible, don't think either will be a problem. Apologies if its only next week though, so much to get through in the next couple of days.